cfde_ap/actions.py

```diff
@@ -49,19 +52,14 @@ def deriva_ingest(servername, archive_url, deriva_webauthn_user,
     header_map = {
         CONFIG['ALLOWED_GCS_HTTPS_HOSTS']: {"Authorization": f"Bearer {https_token}"}
     }
-    try:
-        submission = Submission(server, registry, submission_id, dcc_id, archive_url,
-                                deriva_webauthn_user, archive_headers_map=header_map)
-        submission.ingest()
-    except Exception:
-        # Report the error to Deriva, then raise again to mark the Automate action as a failure.
-        Submission.report_external_ops_error(registry, action_id,
-                                             "Submission raised an error during ingest")
-        raise
+    submission = Submission(server, registry, submission_id, dcc_id, archive_url,
+                            deriva_webauthn_user, archive_headers_map=header_map)
+    submission.ingest()
 
     md = registry.get_datapackage(submission_id)
     return {
-        "success": True,
+        "success": md["status"] == DERIVA_INGEST_SUCCESS,
+        "error": md["diagnostics"],
```

Comment:
To my knowledge, `ingest()` never communicates errors this way, but in case it ever reports errors without raising an exception, this will ensure a 'success' message isn't accidentally emailed to the user.